### PR TITLE
Update dev workflow

### PR DESCRIPTION
**Pull request by ahwagner**
Our current public dev cycle process leaves out important details about intermediate steps by VR leadership and project maintainers, as well as how feature development is tracked. This PR adds those details.

### CODE REVIEW

File: docs/source/appendices/development_process.rst [hunk 1]
```diff
@@ -14,34 +14,57 @@ latest development code on the master branch for community review (see
...
-in our weekly VR calls. These discussions are used to inform whether
-or not a feature will be planned for development. The :ref:`Project
-Leadership <project-leadership>` is responsible for making the final
-determination on whether a feature is to be added to VRS.
+public review and discussion.
+
+Project Leadership Review
+$$$$$$$$$$$$$$$$$$$$$$$$$
+Open issues are reviewed and triaged by the :ref:`Project Leadership
+<project-leadership>`. Feature requests identified to support an unmet
```
[the remark this answers is not in the document]
Comment (ahwagner) [reply docs/source/appendices/development_process.rst:28]: This is just a problem with the GitHub render of `.rst` files. When rendered by Read the Docs (or more generally, by Sphinx), this works fine. Here is the RTD render of this branch: https://vr-spec.readthedocs.io/en/update-dev-workflow/appendices/development_process.html#

File: docs/source/appendices/development_process.rst [hunk 2]
```diff
@@ -14,34 +14,57 @@ latest development code on the master branch for community review (see
...
+need are added to the `Backburner`_ project column and scheduled for
+discussion in our weekly VR calls. These discussions are used to inform
+whether or not a feature will be planned for development. The :ref:`Project
+Maintainers <project-maintainers>` are responsible for making the
```
[the remark this answers is not in the document]
Comment (ahwagner) [reply docs/source/appendices/development_process.rst:32]: This is just a problem with the GitHub render of `.rst` files. When rendered by Read the Docs (or more generally, by Sphinx), this works fine. Here is the RTD render of this branch: https://vr-spec.readthedocs.io/en/update-dev-workflow/appendices/development_process.html#

File: docs/source/appendices/development_process.rst [hunk 3]
```diff
@@ -14,34 +14,57 @@ latest development code on the master branch for community review (see
...
+After community review of requirements, the issue moves on to the
+`In Progress`_ project column. In this stage, the draft features
+will be developed as a draft `Pull Request`_.
+
+Feature Review
+$$$$$$$$$$$$$$
+Once the PR is ready for review, the Pull Request leaves draft
+status and the issue advances to the `QA/Feedback`_ project column.
+Features ready for public review MAY be merged into the master
+branch by pull request through review and approval by at least one
```
[the remark this answers is not in the document]
Comment (ahwagner) [reply docs/source/appendices/development_process.rst]: Done! See https://github.com/ga4gh/vr-spec/pull/221/commits/5196aaecb5bed851e808f279e3fc5c97c667f60b. Thanks for the suggestion!

**Comment by reece** [comment]
This is a huge step in conveying our development process. Thanks @ahwagner. I apologize for the delay in reviewing.

Comments:

* A figure is helpful. I added a fig from a VR call several months ago. However, I wonder whether we should update it to sync better with the text.

* As part of this change, I think we should create a Contacts page (or some such), add info about the mailing list and meetings, and move the Leadership section there.

* Then... drop the Release Cycle heading and promote the other headings so that the whole page is just about the development process. (My concern here is that Development Process and Release Cycle are close enough to the same thing that we shouldn't have two names, but dropping "Release Cycle" right now is awkward with the current heading structure.)

What do you all (@rrfreimuth @larrybabb @ahwagner ) think about these? None are a big deal to me, so if it ships as-is, I'm also totally fine.

**Comment by larrybabb** [comment]
@reece. I'm fine with as-is for now and then incorporating your other suggestions in the next release. I don't think we should delay getting 1.1 out any longer.

**Pull request merged**